ext/search_kit/ang/crmSearchAdmin/searchListing/searchList.html

```diff
@@ -3,7 +3,7 @@ <h1 crm-page-title>{{:: ts('Saved Searches') }}</h1>
 
   <!-- Tabs based on the has_base filter -->
   <div class="crm-search-nav-tabs">
-    <ul class="nav nav-tabs">
+    <ul role="tablist" class="nav nav-tabs">
       <li ng-repeat="tab in $ctrl.tabs" role="presentation" ng-class="{active: $ctrl.tab === tab.name}">
```

Comment:
@colemanw should this role be updated as well? what was role presentation meaning in the first place?

Reply:
Good catch @seamuslee001, I've added it. `role="presentation"` is generic, and so changing it to the more specific `role="tab"` is more helpful to a11y.